### PR TITLE
[WIP] Add person selector

### PR DESCRIPTION
**Pull request by LukasReschke**
This adds a person selector as well as avatars so that initiating a call will be more intuitive. This is WIP and doesn't already do everything.

The idea is basically that by default you just type a name and then once you have completed that join a channel for those two people. The other person will then get a notification (to be done later) and a screen will be shown to the current user indicating that you're waiting until the other user joins the channel.

## TODO

- [x] Join channel if name is selected
- [x] Create channel if name is selected and does not already exists
- [x] Show the other users' name to each other in the menu overview (i.e. the initiator sees "Frank" while Frank sees "Lukas")

To create a group chat the three dot menu is currently being implemented by @Ivansss on another branch.

Fixes https://github.com/nextcloud/spreed/issues/25
Fixes https://github.com/nextcloud/spreed/issues/11

cc @jancborchardt @ChristophWurst FYI

### CODE REVIEW

**Comment by LukasReschke** [comment]
@jancborchardt Needs CSS love…

**Comment by LukasReschke** [comment]
To have this kinda performant https://github.com/nextcloud/server/pull/1158 is required, aka: Use master. Not a stable branch 😉

File: js/app.js [hunk 1]
```diff
@@ -27,19 +27,78 @@
...
+					}
+				},
+				initSelection: function (element, callback) {
+					callback({id: element.val()});
+				},
+				formatResult: function (element) {
+					return '<span><div class="avatar" data-user="' + escapeHTML(element.id) + '" data-user-display-name="' + escapeHTML(element.id) + '"></div>' + escapeHTML(element.id) + '</span>';
+				},
+				formatSelection: function (element) {
+					return '<span><div class="avatar" data-user="' + escapeHTML(element.id) + '" data-user-display-name="' + escapeHTML(element.id) + '"></div>' + escapeHTML(element.id) + '</span>';
```
Comment (LukasReschke) [inline js/app.js]: Guess proper templating helps 🙈

Comment (jancborchardt) [inline js/app.js]: Btw, why is there a div (block-level element) in a span (inline element) ;) Shouldn’t be like that. Just make both elements divs.

**Pull request merged**